akka-cluster-sharding/src/main/scala/akka/cluster/sharding/internal/EventSourcedRememberEntitiesShardStore.scala

```diff
@@ -93,30 +106,21 @@ private[akka] final class EventSourcedRememberEntitiesShardStore(
 
     case RememberEntitiesShardStore.Update(started, stopped) =>
       val events =
-        (if (started.nonEmpty) EntitiesStarted(started) :: Nil else Nil) :::
-        (if (stopped.nonEmpty) EntitiesStopped(stopped) :: Nil else Nil)
+        (if (started.nonEmpty) createEntityEvents(started, EntitiesStarted, maxUpdatesPerWrite) else Nil) :::
+        (if (stopped.nonEmpty) createEntityEvents(stopped, EntitiesStopped, maxUpdatesPerWrite) else Nil)
```

Comment:
Oh, one small thing for Scala 3 compatibility:
```suggestion
        (if (started.nonEmpty) createEntityEvents(started, EntitiesStarted.apply _, maxUpdatesPerWrite) else Nil) :::
        (if (stopped.nonEmpty) createEntityEvents(stopped, EntitiesStopped.apply _, maxUpdatesPerWrite) else Nil)
```

Reply:
Thanks, I was wondering why the Scala 3 compatibility wasn’t passing!